python/src/GaussianProcessFitterResult_doc.i.in

```diff
@@ -0,0 +1,194 @@
+%feature("docstring") OT::GaussianProcessFitterResult
+"Gaussian process fitter result.
+
+Parameters
+----------
+inputSample, outputSample : :class:`~openturns.Sample`
+    The samples :math:`(\vect{x}_k)_{1 \leq k \leq \sampleSize} \in \Rset^{\inputDim}` and :math:`(\vect{y}_k)_{1 \leq k \leq \sampleSize}\in \Rset^{\outputDim}`.
+metaModel : :class:`~openturns.Function`
+    The meta model: :math:`\tilde{\cM}: \Rset^{\inputDim} \rightarrow \Rset^{\outputDim}`, defined in :eq:metaModelGPF.
+residuals : :class:`~openturns.Point`
+    The residual errors.
+relativeErrors : :class:`~openturns.Point`
+    The relative errors.
+basis :  :class:`~openturns.Basis`
+    Functional basis of size :math:`b` : :math:`(\varphi^l: \Rset^{\inputDim} \rightarrow \Rset^{\outputDim})` for each :math:`l \in [1, b]`.
+    Its size should be equal to zero if the trend is not estimated.
+trendCoef : sequence of float
+    The trend coefficients vectors :math:`(\vect{\alpha}^1, \dots, \vect{\alpha}^{\outputDim})` stored as a Point.
+covarianceModel : :class:`~openturns.CovarianceModel`
+    Covariance function of the Gaussian process with its optimized parameters.
+optimalLogLikelihood : float
+    The maximum log-likelihood corresponding to the model.
+linAlgMethod : int
+    The used linear algebra method to fit the model.
+    0 for `LAPACK`, 1 for `HMAT`
+
+Notes
+-----
+The structure is usually created by the method *run()* of a :class:`~openturns.experimental.GaussianProcessFitter`, and obtained thanks to the *getResult()* method.
+
+The meta model :math:`\tilde{\cM}: \Rset^{\inputDim} \rightarrow \Rset^{\outputDim}` is defined by:
+
+.. math::
+    :label: metaModelGPF
+
+    \tilde{\cM}(\vect{x}) = \left(
+      \begin{array}{l}
+        \mu_1(\vect{x}) \\
+        \dots  \\
+        \mu_p(\vect{x}) 
+       \end{array}
+     \right)
+
+where :math:`\mu_l(\vect{x}) = \sum_{j=1}^{b} \alpha_j^l \varphi_j^l(\vect{x})` and :math:`\varphi_j^l: \Rset^{\inputDim} \rightarrow \Rset` are the trend functions 
```

Comment:
Please split this line and make it less than 79 characters, so that the review is easier.